pkg/print/list/base.go

```diff
@@ -16,8 +16,17 @@ type Formatter interface {
 	FormatStatusEvent(se event.StatusEvent) error
 	FormatPruneEvent(pe event.PruneEvent) error
 	FormatDeleteEvent(de event.DeleteEvent) error
+	FormatWaitEvent(we event.WaitEvent) error
 	FormatErrorEvent(ee event.ErrorEvent) error
-	FormatActionGroupEvent(age event.ActionGroupEvent, ags []event.ActionGroup, as *ApplyStats, ps *PruneStats, ds *DeleteStats, c Collector) error
+	FormatActionGroupEvent(
```

Comment:
I am not sure it is necessary to put every parameter into a separate line.

Reply:
The conventional patterns are to either use a single line or one line per argument. But the line length convention in this repo wraps and creates something in between the two conventions. I tend to bias towards line per argument when reaching the character limit.

Reply:
Looks like Google has conventions for this, but I don't know if there's a public equivalent: go/go-style/decisions#func-formatting

Reply:
Unfortunately, the Google conventions don't use an 80 character max line (which this repo sort of does). So the conventions on when to wrap and when not to wrap don't cleanly apply.

Reply:
Agreed. The stats objects should probably all be wrapped into a single object, but it didn't seem particularly high priority.